jax/experimental/jax2tf/tests/primitives_test.py

```diff
@@ -461,6 +455,15 @@ def test_stop_gradient(self):
     f = jax2tf.convert(lax.stop_gradient)
     self.assertEqual(f(tf.ones([])), 1.)
 
+  # test_bfloat16 checkes that https://github.com/google/jax/issues/3942 is fixed
+  def test_bfloat16(self):
```

Comment:
Perhaps we can rename this `test_bfloat16_constant`

Reply:
Addressed